tests/test_enterprise/api/test_views.py

```diff
@@ -1103,3 +1199,56 @@ def test_enterprise_catalog_courses_unauthorized(self):
         assert response_content['detail'] == 'User {username} is not associated with an EnterpriseCustomer.'.format(
             username=self.user.username
         )
+
+    @ddt.data(
+        (False, {}, None),
+        (False, {'course_run_ids': ['fake1', 'fake2']}, None),
+        (
+            True,
+            {
+                'course_run_ids': [
+                    fake_catalog_api.FAKE_COURSE_RUN['key'],
+                    fake_catalog_api.FAKE_COURSE_RUN2['key']
+                ]
+            },
+            [fake_catalog_api.FAKE_COURSE_RUN, fake_catalog_api.FAKE_COURSE_RUN2]
+        ),
+        (
+            True,
+            {
+                'program_uuids': [
+                    fake_catalog_api.FAKE_PROGRAM_RESPONSE1['uuid'],
+                    fake_catalog_api.FAKE_PROGRAM_RESPONSE2['uuid']
+                ]
+            },
+            [fake_catalog_api.FAKE_PROGRAM_RESPONSE1, fake_catalog_api.FAKE_PROGRAM_RESPONSE2]
+        ),
+    )
+    @ddt.unpack
+    @mock.patch('enterprise.models.CourseCatalogApiServiceClient')
+    def test_enterprise_customer_contains_content(self, contains_content, query_params, search_results,
+                                                  mock_catalog_api_client):
+        """
+        The ``programs`` detail endpoint should return correct results from course discovery,
+        with enterprise context in courses.
```

Comment:
@douglashall Copy-paste leftover? This doesn't seem to describe what the test does :)

Reply:
Yup.